rusoto/core/src/request.rs

```diff
@@ -170,21 +198,43 @@ impl fmt::Display for TlsError {
     }
 }
 
+/// TODO: docs
+pub struct DefaultTlsClient(Reactor);
```

Comment:
Small nitpick but I prefer using:

```
pub struct DefaultTlsClient {
  reactor: Reactor,
}
```

This not only makes it easier to add on into the future, but imo removes the hard to read: `self.0.`

Reply:
Sure, no prob!

- [ ] change tuple structs to proper structs with named fields